feature-libs/quote/components/actions/confirm-dialog/quote-actions-confirm-dialog.component.html

```diff
@@ -83,5 +83,16 @@
         </button>
       </div>
     </div>
+
+    <!-- For screen reader purposes (not visual)-->
+    <div
+      class="cx-visually-hidden"
+      aria-atomic="true"
+      aria-live="assertive"
+      role="alert"
+      aria-relevant="additions"
+    >
+      {{ confirmationContext.a11y.title | cxTranslate }}
```

Comment:
I would provide the detailed text to the screenreader user. I.e. for cancel-modal " Are you sure you want to cancel this quote request?"
Furthermore if a confirm text and a warning message is displayed. Both should be included.